gpt4all-chat/modellist.cpp

```diff
@@ -1502,9 +1503,94 @@ void ModelList::parseModelsJsonFile(const QByteArray &jsonData, bool save)
             { ModelList::ParametersRole, "?" },
             { ModelList::QuantRole, "NA" },
             { ModelList::TypeRole, "GPT" },
+            { ModelList::UrlRole, "https://api.openai.com/v1/chat/completions"},
         };
         updateData(id, data);
+        
     }
```

Comment:
NOTE: In future we'll have a separate category/space on the download dialog that caters to these types of models...

Reply:
Not sure I see the need? These all require the user to proactively - install - them by adding api keys, right?

Reply:
I thought you wanted to separate offline models from onlines ones

Reply:
Yes, but I'm saying I plan on handling this in a different way with a complete redesign of the 'download dialog' that will separate things into categories ;)